packages/expect/src/__tests__/matchers.test.js

```diff
@@ -894,6 +894,28 @@ describe('.toBeCloseTo()', () => {
     });
   });
 
+  [[Infinity, Infinity], [-Infinity, -Infinity]].forEach(([n1, n2]) => {
```

Comment:
Great tests 👌

Reply:
Thank you, working on that CHANGELOG.md as I speak. New to open source contributions, hope im doing it right!